src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Stats/GetAzureExpressRouteCircuitRouteTableSummaryCommand.cs

```diff
@@ -0,0 +1,91 @@
+// You may obtain a copy of the License at		
+// http://www.apache.org/licenses/LICENSE2.0		
+// Unless required by applicable law or agreed to in writing, software		
+// distributed under the License is distributed on an "AS IS" BASIS,		
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.		
+// See the License for the specific language governing permissions and		
+// limitations under the License.		
+// 		
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Management.Automation;
+using AutoMapper;
+using Microsoft.Azure.Commands.Tags.Model;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Commands.Network.Models;
+using MNM = Microsoft.Azure.Management.Network.Models;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    using System;
+    using System.Collections;
+    using System.Linq;
+
+    using AutoMapper;
+
+    [Cmdlet(VerbsCommon.Get, "AzureRmExpressRouteCircuitRouteTableSummary"), OutputType(typeof(PSExpressRouteCircuitRoutesTableSummary))]
+    public class GetAzureRmExpressRouteCircuitRouteTableSummaryCommand : NetworkBaseCmdlet
+    {
+        [Alias("ResourceName")]
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The resource name.")]
+        [ValidateNotNullOrEmpty]
+        public virtual string Name { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The resource group name.")]
+        [ValidateNotNullOrEmpty]
+        public virtual string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The Name of ExpressRoute Circuit")]
+        [ValidateNotNullOrEmpty]
+        public string ExpressRouteCircuitName { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The PeeringType")]
+        [ValidateSet(
+           MNM.ExpressRouteCircuitPeeringType.AzurePrivatePeering,
+           MNM.ExpressRouteCircuitPeeringType.AzurePublicPeering,
+           MNM.ExpressRouteCircuitPeeringType.MicrosoftPeering,
+           IgnoreCase = true)]
+        public string PeeringType { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The DevicePath, can be either Primary or Secondary")]
+        [ValidateNotNullOrEmpty]
+        public string DevicePath { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            base.ExecuteCmdlet();
+            DevicePathEnum path;
+            if (Enum.TryParse(DevicePath, true, out path))
+            {
+                var arpTables = this.NetworkClient.NetworkManagementClient.ExpressRouteCircuits.ListRoutesTableSummary(ResourceGroupName, ExpressRouteCircuitName, PeeringType, DevicePath).Value.Cast<object>().ToList();
```

Comment:
this line is too long. please restructure the code in a way that it does not exceed ~120 symbols per line.